docs/sdk-and-tools/indices/accountsesdt.md

```diff
@@ -0,0 +1,74 @@
+---
+id: es-index-accountsesdt
+title: Elasticsearch index: accountsesdt
+---
+
+The `_id` field of this index is composed in this way: `{bech32address}_{tokenIdentifier}_{nonce}` (example: `erd.._abcd-0123-01`).
+
+| Field       | Description                                                                                                                           |
+|-------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| identifier  | The identifier field is composed from the `token` field and the `nonce` field hex encoded.                                            |
```

Comment:
```suggestion
| identifier  | The identifier field consists of `token` field and the `nonce` field hex encoded.                                            |
```
Also, could you add an example here?

Reply:
I think an example would be good here. How exactly are `token` and `nonce` merged for example ("ALICE-nonce"/ "ALICEnonce")?

Reply:
added an example